oak_client/src/attestation.rs

```diff
@@ -84,7 +84,7 @@ pub fn create_tls_attestation_config(
     config.set_protocols(&[b"http/1.1".to_vec(), b"h2".to_vec()]);
 
     // Add root TLS certificate.
-    let mut cc_reader = std::io::BufReader::new(&root_tls_certificate[..]);
+    let mut cc_reader = std::io::BufReader::new(root_tls_certificate);
```

Comment:
Out of curiosity, what was the linting error for this?

Reply:
The error was:

use the original value instead: `root_tls_certificate` 
= note: `-D clippy::redundant-slicing` implied by `-D warnings`